src/lib/AnnotationControls.ts

```diff
@@ -185,6 +186,12 @@ export default class AnnotationControls {
             groupElement,
         );
 
+        groupElement.setAttribute('data-resin-feature', 'annotations');
+        regionButton.setAttribute('data-resin-target', 'highlightRegion');
+        if (fileId) {
```

Comment:
Why is this an optional parameter? What scenario would lead this value to be undefined/null? Is that scenario handled?